corehq/apps/case_importer/templates/case_importer/import_cases.html

```diff
@@ -4,11 +4,23 @@
 
 {% block js-inline %}
 <script id="caseUploadsResource" type="text/plain">{% url 'case_importer_uploads' domain %}</script>
+<script id="caseUploadFileDownloadResource" type="text/plain">{% url 'case_importer_upload_file_download' domain '---' %}</script>
+<script id="caseUploadPermissionDeniedTitle" type="text/plain">{% trans 'You may not download this file yourself' %}</script>
+<script id="caseUploadPermissionDeniedContent" type="text/plain">{% trans 'Please ask an administrator or the user who uploaded the file for their help' %}</script>
```

Comment:
I changed this in https://github.com/dimagi/commcare-hq/compare/case-import-history-phase2...case-import-history-phase-1-feedback as well, but not to that.

FWIW the reason I didn't go with

```
var caseUploadPermissionDeniedTitle = '{% trans "some text" %}';
```

is because it's not safe to dump arbitrary text into a string-in-javascript(-in-html) context, both just pedantically, and also in the specific case of translations, an `'` is actually a legitimate concern.

```
var iLoveThis = '{% trans "I love this!" %}';
```

would be rendered in French as

```
var iLoveThis = 'J'adore ça!';
```

which is a javascript syntax error.